packages/jaeger-ui/src/components/SearchTracePage/SearchForm.test.js

```diff
@@ -392,6 +392,19 @@ describe('<SearchForm>', () => {
     btn = wrapper.find(`[data-test="${markers.SUBMIT_BTN}"]`);
     expect(btn.prop('disabled')).toBeTruthy();
   });
+
+  it('uses config.search.maxLimit', () => {
+    const maxLimit = Math.floor(Math.random() * 7000) + 3000;
```

Comment:
a fixed constant would do just as well, e.g. 1234 (unlikely to be the default value elsewhere)

Reply:
Ok, I'll use a constant.